README.md

```diff
@@ -105,6 +105,30 @@ folder structure as per above and modify the buildspec.yml file to specify the v
                        --py_versions py3
     ```
 
+### Building your image using Docker CLI
```

Comment:
Note: if we add this to our README, we'll have to support questions/issues associated with building images directly through docker CLI. It will be worth it to call out (in a note) that this is not a regularly tested operation

Reply:
Done. Please take a look at the "Note:" section.